samples/system-test/test.samples.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const assert = require('assert');
+
+const createJob = require('../createJob');
```

Comment:
Instead of exporting the methods and testing them like unit tests, we've been trying to do more and more complete e2es that use `execa` to actually call the program and run it.  Would it be possible to go this route?

Reply:
Does this mean adding a simple CLI in order to run this function? I was trying to stay away from adding a CLI since it isn't needed for a how-to snippet.

Reply:
It's really easy!  You don't need to mess around with yargs or anything.  You can just do:

```js
const args = process.argv.slice(2);
createJob(...args).catch(console.error);
```

That's it :) Now running `node createJob.js` will "just work".

Reply:
done.